gradlew

```diff
@@ -1,25 +1,59 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
+
+#
+# Copyright 2015 the original author or authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
 
 ##############################################################################
 ##
 ##  Gradle start up script for UN*X
```

Comment:
This doesn't appear changed but surprises me. AFAIK HTA has never been compiled on a Unix/Linux system. I've always built it with Android Studio on Windows. So maybe this file isn't used at all? Or maybe Android Studio has a way to run Unix scripts?